lib/matplotlib/figure.py

```diff
@@ -1596,25 +1596,37 @@ def subplots(self, nrows=1, ncols=1, sharex=False, sharey=False,
             return axarr
 
     def _remove_ax(self, ax):
-        def _reset_loc_form(axis):
+        def _reset_loc_from(axis):
```

Comment:
Why did you change the name?

Reply:
Because the function is resetting locators from another axis, hence the shortening `reset_loc_from`.

Reply:
I think it’s reset locators formatters

Reply:
should better be `_reset_locators_and_formatters`.